core/storage/question/gae_models_test.py

```diff
@@ -199,3 +199,31 @@ def test_get_models_by_question_id(self):
                 'question_id3')
         )
         self.assertEqual(len(question_skill_links), 0)
+
+    def test_get_question_skill_links_with_constant_number_per_skill(self):
```

Comment:
Test a case when we ask for 4 questions (I think based on the current logic it will return 3 right? 2 from skill_id1 and 1 from skill_id3)

Reply:
Done.